sdk/core/core-http/test/serializationTests.ts

```diff
@@ -286,9 +286,9 @@ describe("msrest", function() {
         required: false,
         serializedName: "DateTimeRfc1123"
       };
-      const rfc = new Date("Mon, 01 Jan 0001 00:00:00 GMT");
+      const rfc = new Date("Thu, 01 Jan 3001 00:00:00 GMT");
```

Comment:
bizarre to me that we're worried about times thousands of years from the present

Reply:
yeah, I will update to some date in 2020